homework/src/index.js

```diff
@@ -1,47 +1,103 @@
-'use strict';
+"use strick";
```

Comment:
This is a silly typing error to make (and flagged by the spelling checker). Your code is now not using strict mode.

Reply:
I can see all you have mentioned and I will fix them and come back again! thanks for your comments